vochain/transaction.go

```diff
@@ -585,3 +597,43 @@ func checkRevealProcessKeys(tx *models.AdminTx, process *models.Process) error {
 	}
 	return nil
 }
+
+func SetTransactionCostsTxCheck(vtx *models.Tx, txBytes, signature []byte, state *State) (uint64, error) {
+	tx := vtx.GetSetTransactionCosts()
+	// check signature available
+	if signature == nil || tx == nil || txBytes == nil {
+		return 0, fmt.Errorf("missing signature and/or transaction")
+	}
+	// check value
+	if tx.Value <= 0 {
```

Comment:
This check removes the possibility to declare a tx with cost 0, I think we should allow that